### PR TITLE
GOB: Add detection for ADI Jnr. (Amiga)

### PR DESCRIPTION
**Pull request by einstein95**
The title screen uses "Jnr." as the abbreviation, so I've used it here

### CODE REVIEW

**Comment by bluegr** [comment]
What is the status of the game? Is it working fully, or are there issues?

**Comment by einstein95** [comment]
> What is the status of the game? Is it working fully, or are there issues?

It runs fine, more or less, with only a couple major issues

image
After getting past the copy protection screen, this empty box appears and a few warnings in console
```
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: File "DISK2.STK" not found!
```
In UAE, this appears as
image


image
Attempting to enter the house gives you this screen and these console warnings (the last line being when you exit the screen)
```
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: Attempted to write to file "bou.inf"!
WARNING: STUB: Adibou1 o2_gob0x0B!
WARNING: STUB: Adibou1 o2_gob1001!
```
In UAE, this instead gives an "Insert Application" error
image

Would this be enough to downgrade it to unstable?

**Comment by sev-** [comment]
Yes, unstable please

**Comment by CrystalBen** [comment]
Hello, i have the German Version of Adi Junior for Windows 3.1 can anyone help me to get running under scummvm?

**Comment by sdelamarre** [comment]
> Yes, unstable please
I worked a little on it, I confirm that the state should be "unstable". Besides the read/count applications not being reachable (probably due to the stubbed opcodes listed above in the log), the game occasionaly crashes, e.g. when using some tools in the "paint" activity.

**Comment by tag2015** [comment]
There is also a report [13972](https://bugs.scummvm.org/ticket/13972) for the DOS UK version

Matched game IDs for the gob engine: ajworld
{"intro.stk", 0, "6d234641b74b3bdf746c39a64ff1abcc", 2678326},

File: engines/gob/detection/tables_ajworld.h [hunk 1]
```diff
@@ -41,4 +41,22 @@
...
+// -- Amiga Floppy --
+
+{
+	{
+		"ajworld",
+		"",
+		AD_ENTRY1s("intro.stk", "71e7db034890885ac96dd1be43a21c38", 556834),
+		EN_ANY,
+		kPlatformAmiga,
+		ADGF_NO_FLAGS,
```
Comment (bluegr) [inline engines/gob/detection/tables_ajworld.h:54]:
```suggestion
		ADGF_UNSTABLE,
```

**Comment by bluegr** [comment]
Given the feedback for this game, I've marked it as unstable. Merging

**Pull request merged**